roles/foreman_provisioning/tasks/configure_centos_9.yml

```
@@ -0,0 +1,27 @@
+- name: "Ensure CentOS Stream 9"
+  theforeman.foreman.operatingsystem:
+    username: "{{ foreman_provisioning_username }}"
+    password: "{{ foreman_provisioning_password }}"
+    server_url: "{{ foreman_provisioning_server_url }}"
+    name: CentOS Stream
+    family: Redhat
+    major: 9
+    architectures:
+      - x86_64
+    media:
+      - CentOS Stream 9 mirror
+    provisioning_templates:
+      - Kickstart default
+    ptables:
+      - Kickstart default
+    state: present
+
+- name: "Set default template for CentOS Stream 9"
+  theforeman.foreman.os_default_template:
+    username: "{{ foreman_provisioning_username }}"
+    password: "{{ foreman_provisioning_password }}"
+    server_url: "{{ foreman_provisioning_server_url }}"
+    operatingsystem: "CentOS Stream 9"
+    template_kind: "kickstart"
```

```suggestion
    template_kind: "provision"
```

The linter is right there is no "kickstart" kind, just "provision"